### PR TITLE
AAU, on the SDK, when I call kili.get_project, I instantiate a project object with high level methods

### PR DESCRIPTION
**Pull request by theodu**
…e client. Move exceptions in a specific file

- [ ] I opened a merge request on `kili` on a branch with the same name than the branch of the current pull request and forced security tests.
- [ ] I opened a pull request on the Node playground if the changes are breaking.

### CODE REVIEW

File: test/test_client_project.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    """
+
+    def test_get_unexisting_project(self):
+        """Test that get_project on a unexisting project_id raises a NotFound Error."""
+        with pytest.raises(NotFound):
+            kili.get_project('this_project_id_do_not_exist')
+
+    def test_get_existing_project(self, create_video_project):
+        """Test that get_project on an existing project_id return a Project with the right id."""
+        project_id = create_video_project
```
Comment (PierreLeveau) [inline test/test_client_project.py:30]: I'm against keeping digging the technical debt of testing on project that are hosted on the cloud. Can you mock the `kili.projects` method, as it is done in AutoML ?

**Review by PierreLeveau — CHANGES_REQUESTED** [review]

File: test/test_client_project.py (hunk 1 above)
Comment (theodu) [reply test/test_client_project.py:30]: Yes that's true, I'll change that

File: kili/client.py [hunk 2]
```diff
@@ -75,3 +77,23 @@ def __init__(self, api_key=os.getenv('KILI_API_KEY'),
...
+        """Return a project object corresponding to the project_id given.
+        The returned project object inherit from many methods for project management
+
+        Args:
+            project_id: id of the project to return
+
+        raise:
+            NotFound if the given `project_id` does not correspond to an existing project
+        """
+        query = self.projects(project_id=project_id,
```
Comment (mDuval1) [inline kili/client.py:91]: this is not a query but a response

File: kili/helpers.py [hunk 3]
```diff
@@ -13,6 +13,8 @@
 
 import requests
 
+from kili.exceptions import EndpointCompatibilityError, GraphQLError
```
Comment (mDuval1) [inline kili/helpers.py:16]: good to split in coherent files

File: kili/project/__init__.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
+"""Project module."""
+
+
+class Project:  # pylint: disable=too-few-public-methods
+    """
+    Object that represents a project in the database.
```
Comment (mDuval1) [inline kili/project/__init__.py:6]: NOT `in the database` (not a concern of DS), but in `Kili`

**Review by PierreLeveau — APPROVED** [review]

File: kili/project/__init__.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
+"""Project module."""
+
+
+class Project:  # pylint: disable=too-few-public-methods
```
Comment (mDuval1) [inline kili/project/__init__.py:4]: I don't think you should directly have a folder "project" at the root, but it depends on what you want to do. Do we intend to have a folder dataset, asset, label, etc... ? If so then we should have a folder above, named `entities` or something along those lines

Comment (theodu) [reply kili/project/__init__.py:4]: That is what has been seen with Pierre and Edouard. It comes into the API v3 that we would like to deliver in the next quarter with a better structure, better UX and better function names. one idea would be to have several modules, one with project for example, to work on a specific project, one for automl later, one for managing its organization etc. This creates better scaling of the SDK and allows to work more easily with projects.
A more precise spec will come later, after the CLI
```
project = kili.get_project('<project_id>')
project.import(files)
```

**Pull request merged**